base/doc/ltnews41.tex

```diff
@@ -417,6 +417,15 @@ \subsection{Process global options once per package}
 %
 \githubissue{1619}
 
+\section{Documentation}
+\subsection{Clarifying space handling of \cs{textcolor}}
+
+In contrast to other \cs{text}-commands like \cs{textbf} or \cs{textrm},
+the command \cs{textcolor} gobbles spaces at the start of its argument, so \verb=Hello\textcolor{red}{  World}= will output Hello\textcolor{red}{  World}.
+This behaviour has now been clarified in the documentation.
```

Comment:
There are technical as well as compatibility reasons for this, so the behavior will not change. This has now been clarified ...